android/tangram/src/com/mapzen/tangram/HttpHandler.java

```diff
@@ -1,24 +1,39 @@
 package com.mapzen.tangram;
 
-import com.squareup.okhttp.Cache;
-import com.squareup.okhttp.Callback;
-import com.squareup.okhttp.OkHttpClient;
-import com.squareup.okhttp.Request;
+import okhttp3.Cache;
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
 
 import java.io.File;
-import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
 import java.util.concurrent.TimeUnit;
 
 public class HttpHandler {
 
     private OkHttpClient okClient;
+    private OkHttpClient.Builder okClientBuilder;
     private Request.Builder okRequestBuilder;
 
+    class CallEntry {
+        CallEntry(String url, Call call) {
+            this.url = url;
+            this.call = call;
+        }
+        public String url;
+        public Call call;
+    }
+
+    private ArrayList<CallEntry> calls = new ArrayList<>();
```

Comment:
okhttp3 dispatcher provides `queuedCalls` and `runningCalls` API, so we do not have to maintain a list on our own.

Refer: https://square.github.io/okhttp/3.x/okhttp/okhttp3/Dispatcher.html